src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -354,6 +354,7 @@ class MoneyRequestAmountPage extends React.Component {
                         pressOnEnter
                         isDisabled={!this.state.amount.length || parseFloat(this.state.amount) < 0.01}
                         text={this.props.buttonText}
+                        enterKeyEventListenerPriority={2}
```

Comment:
@azimgd Why do we need to set the priority to `2`? Is `1` will be enough since there's not `Enter` key listener in the `MoneyRequestAmountPage`?

Reply:
Decreased.